Gopkg.lock

```diff
@@ -797,10 +784,12 @@
   revision = "eb3733d160e74a9c7e442f435eb3bea458e1d19f"
 
 [[projects]]
-  branch = "release-1.8"
```

Comment:
FYI, I'll be bumping these to `version = "kubernetes-1.9.1"`

Reply:
Why not the latest stable in the 1.9.x branch?

Reply:
Isn't release more "stable" than HEAD on a branch? I'm not sure that I care, just that the client-go package is using branch = "release-6.0" which isn't clear which k8s version it supports.  Tagging based on release just seems more straight forward and definitive.